### ISSUE TITLE
`grid: false` does not disable adding of `-ms-grid` to `@supports`

### ISSUE THREAD

**Issue by sgomes** [postcss/autoprefixer#778]
The following input CSS:

```css
@supports (display: grid) {
  .foo {
    display: grid;
  }
}
```

produces the following result with the `grid: false` option passed to autoprefixer:

```css
@supports ((display: -ms-grid) or (display: grid)) {
  .foo {
    display: grid; } }
```

This is unexpected and leads to inconsistent prefixing. Setting `grid: false` should in my mind disable it everywhere, including `@supports` rules.

**Comment by ai** [comment]
Sure. Thanks for report. I will try to fix it today.

**Comment by sgomes** [comment]
Thanks for the super-speedy reply, @ai!

**Comment by ai** [comment]
Done 7a4350a

**Comment by ai** [comment]
Released in 6.7.2.

**Comment by sgomes** [comment]
That was incredibly quick, thank you, @ai! Moving to 6.7.2 as the minimum version in my code, and removing the workaround :)
